static/app/components/smartSearchBar/index.tsx

```diff
@@ -72,6 +72,8 @@ const ACTION_OVERFLOW_STEPS = 75;
 
 const makeQueryState = (query: string) => ({
   query,
+  // Anytime the query changes and it is not "" the dropdown should show
+  showDropdown: !!query,
```

Comment:
We should have the dropdown show on the empty query so we can show the default state. So I guess have it show every time the query is changed instead?

```suggestion
  showDropdown: true,
```

Reply:
lol, I actually did this change after demoing to @Zylphrex, but havent pushed it. Good eye!